rules/azure/persistence_azure_pim_user_added_global_admin.toml

```
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/09/24"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an Azure Active Directory (AD) Privileged Identity Management (PIM) user addition to the Global Administrator
+role. PIM is a service that enables you to manage, control, and monitor access to important resources in an
+organization. Users who are assigned to the Global administrator role can read and modify any administrative setting
+in your Azure AD organization.
+"""
+false_positives = [
+    """
+    Global administrator additions may be done by a system or network administrator. Verify whether the username,
+    hostname, and/or resource name should be making changes in your environment. Global administrator additions from
+    unfamiliar users or hosts should be investigated. If known behavior is causing false positives, it can be exempted
+    from the rule.
+    """,
+]
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Azure PIM User Addition to Global Administrator Role"
```

I think it would be more accurate to say

```suggestion
name = "Azure Global Administrator Role Added to PIM User"
```

For this and the description